test/pathsDB.js

```diff
@@ -12,13 +12,13 @@ contract('PathsDB', (accounts) => {
 
   it ("returns an array of all checkpoints addresses", () => {
     return PathsDB.deployed().then((instance) => {
-      instance.addCheckpoint(accounts[0]);
-      instance.addCheckpoint(accounts[1]);
+      instance.addCheckpoint(accounts[8]);
```

Comment:
It passed 3 times in a row before I pushed but I'm still not super confident about it.